pyriemann/datasets/simulated.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
+from sklearn.utils.validation import check_random_state
+from .sampling import generate_random_spd_matrix, sample_gaussian_spd
+from pyriemann.utils.distance import distance_riemann
+from pyriemann.utils.base import powm, sqrtm
+
+
+def make_gaussian_blobs(n_samples=100, n_dim=2, class_sep=1.0, class_disp=1.0,
+                        random_state=None):
+    """Generate SPD dataset with two classes sampled from Riemannian Gaussian
+
+    Generate a dataset with SPD matrices generated from two Riemannian
+    Gaussian distributions. The distributions have the same class dispersions
+    and the distance between their centers of mass is an input parameter.
+
+    Parameters
+    ----------
+    n_samples : int (default: 100)
```

Comment:
The ambiguous term `samples` is back... 

In our discussions, we have mentionned `n_epochs` to replace `n_trials`, but the term `epochs` is clearly not appropriate here.

What about `n_mats`? It seems generic enough to replace `n_trials` everywhere in pyRiemann, and to replace `n_samples` in this module.
@plcrodrigues @sylvchev @agramfort @lkorczowski

Reply:
indeed....

n_matrix?

we need to sleep over it....

Reply:
Completely agree! why not `n_matrices`?

Reply:
yeah, `n_matrices` sounds good to me